install.md

```diff
@@ -203,3 +203,42 @@ Finally, after the binary and documentation is built:
 ```bash
 sudo make install
 ```
+
+### Building a static binary
+
+There has been efforts in the past to produce and maintain static builds, but the maintainers prefer to run Skopeo using distro packages or within containers. This is because static builds of Skopeo tend to be unreliable and functionally restricted. Specifically:
+- some features of Skopeo depend on non-Go libraries like `libgpgme` and `livdevmapper`,
+- generating static Go binaries uses native Go libraries, which don't support e.g. `.local` or LDAP-based name resolution.
+
+That being said, if you would like to build Skopeo statically, you might be able to do it by:
+- exporting environment variable `CGO_ENABLED=0` (disabling CGO causes Go to prefer native libraries when possible, instead of dynamically linking against system libraries)
+- passing the `BUILDTAGS=containers_image_openpgp` Make flag (this remove the dependency on `libgpgme` and its companion libraries)
+- commenting out the `GO_DYN_FLAGS` line in the Makefile (this flag seems to force the creation of a dynamic executable)
+
+The following command implements these steps to produce a static binary in the `bin` subdirectory of the repository:
+
+```bash
+sed 's/GO_DYN_FLAGS=/#/' Makefile |
```

Comment:
Hum, (assuming the other question doesn’t makes this moot), I’d rather have this as a `Makefile` variable or target instead of a document for how to edit the `Makefile`; this could extremely easily break without anyone noticing.

---

In my testing, just `make … GO_DYN_FLAGS=` seems to work fine to override the variable, just like `make BUILDTAGS=…` works. Does that work for you?